### PR TITLE
Add pricing tier checks, next-seo, and analytics.

### PR DESCRIPTION
**Pull request by leerob**
- Added pricing checks based on the Stripe claim on the JWT (starter, premium)
- Added `next-seo` for setting up some defaults for meta/OG tags.
- Added Fathom for analytics

### CODE REVIEW

**Comment by leerob** [comment]
#### 📝Changed routes:


| [**/account**](https://fastfeedback-git-stream-10.leerob1.now.sh/account) | [**/dashboard**](https://fastfeedback-git-stream-10.leerob1.now.sh/dashboard) |
|:-:|:-:|
|<a href="https://fastfeedback-git-stream-10.leerob1.now.sh/account">Screenshot of /account</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/_CmW7gx5xuUrUfboCjc0vd3NZjM_.png">(view full size)</a> |<a href="https://fastfeedback-git-stream-10.leerob1.now.sh/dashboard">Screenshot of /dashboard</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Hxx6g3Enq582PAw7D7ea7d6SceE_.png">(view full size)</a> |



| [**/feedback**](https://fastfeedback-git-stream-10.leerob1.now.sh/feedback) | [**/**](https://fastfeedback-git-stream-10.leerob1.now.sh/) |
|:-:|:-:|
|<a href="https://fastfeedback-git-stream-10.leerob1.now.sh/feedback">Screenshot of /feedback</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/iyepwUWVjmYSuwUkEmMm1jDdwd4_.png">(view full size)</a> |<a href="https://fastfeedback-git-stream-10.leerob1.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/13rbrM778s_eA2bl_16H_WjfeeI_.png">(view full size)</a> |







Commit a9cd79195a44ac69a48c0e32b31e51224d606705 (https://fastfeedback-git-stream-10.leerob1.now.sh).

File: lib/auth.js [hunk 1]
```diff
@@ -81,13 +81,21 @@ function useProvideAuth() {
...
+const formatUser = async (user) => {
   return {
     uid: user.uid,
     email: user.email,
     name: user.displayName,
     token: user.xa,
     provider: user.providerData[0].providerId,
-    photoUrl: user.photoURL
+    photoUrl: user.photoURL,
+    stripeRole: await getStripeRole()
```
Comment (joe-bell) [inline lib/auth.js:99]: Just a heads up for anyone else who had problems accessing `.xa` to get the token, you should be able to use this approach to get both the `token` and `stripeRole`:

```js
const formatUser = async (user) => {
  const { token, claims } = await user.getIdTokenResult();

  return {
    uid: user.uid,
    email: user.email,
    name: user.displayName,
    token,
    provider: user.providerData[0].providerId,
    photoUrl: user.photoURL,
    stripeRole: claims.stripeRole || "free",
  };
};
```

Comment (leerob) [reply lib/auth.js:99]: Related to this? https://github.com/leerob/fastfeedback/issues/25

Comment (joe-bell) [reply lib/auth.js:99]: Yes! Just extending it to include the Stripe role too

**Pull request merged**